chatez/src/pages/SignUp/SignUp.tsx

```diff
@@ -1,9 +1,125 @@
+
+  const history = useHistory();
+
+  useEffect(() => {
+    setDisplayPhoto(
+      'https://i.pinimg.com/736x/3d/cd/4a/3dcd4af5bc9e06d36305984730ab7888.jpg'
+    );
+  }, []);
+
+  const GotoSignUpPage = () => {
+    history.push('/');
+  };
+
+  const onGoogleSignIn = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!isSigningIn) {
+      setIsSigningIn(true);
+      doSignInWithGoogle().catch((error) => {
+        setIsSigningIn(false);
+        console.error('Error signing in with Google:', error);
+      });
+    }
+  };
+
+  const updateDisplayPhoto = () => {
+    // Trigger the file input click event programmatically
+    if (fileInputRef.current) {
+      fileInputRef.current.click();
+    }
+  };
+
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0]; // Get the selected file
+    if (file) {
+      const fileURL = URL.createObjectURL(file); // Create a preview URL
+      setDisplayPhoto(fileURL); // Update the display photo
+    }
+  };
+
   return (
     <div className="SignUpPage">
-      <h1>Sign Up Page</h1>
+      <div className="SignUpPage-form">
+        <div className="signUp-header">
+          <img id="Logo" src={Logo} alt="Logo" />
+        </div>
+        <form className="SignUp-form">
+          <div className="photo-container">
+            <img id="displayPhoto" src={displayPhoto} alt="Display url" />
+            <span className="camera-icon">
+              <IconButton onClick={updateDisplayPhoto}>
+                <CameraAlt id="camera-button" />
+              </IconButton>
+            </span>
+            <input
+              type="file"
+              ref={fileInputRef}
+              style={{ display: 'none' }}
+              accept="image/*"
+              onChange={handleFileChange}
+            />
+          </div>
+          <input
+            type="text"
+            name="username"
+            placeholder="Username"
+            required
+            value={username}
+            onChange={(e) => setUsername(e.target.value)}
+          />
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="Email Address"
+            required
+            value={email}
+            onChange={(e) => setEmail(e.target.value)}
+          />
+          <input
+            type="password"
+            id="password"
+            name="password"
+            placeholder="Password"
+            required
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
+          />
+          <button type="submit" className="submit-button">
+            Sign Up
+          </button>
+        </form>
+      </div>
+      <div className="SignInWithGoogle-container">
+        <h2>
+          Already have an account?{' '}
+          <span>
+            <button id="Login-button" onClick={GotoSignUpPage}>
+              <u>Log In</u>
```

Comment:
change log in to sign in. normally I think you use create an account if we wanna use log in. but it sign up its sign in. correct me if I am wrong.